PinballY/PlayfieldView.cpp

```diff
@@ -12164,8 +12169,10 @@ void PlayfieldView::SyncInfoBox()
 			infoBox.sprite.Attach(new Sprite());
 			infoBox.sprite->Load(width, height, Draw, eh, _T("Info Box"));
 
-			// move it up towards the top of the screen
-			infoBox.sprite->offset.y = 0.25f;
+			// move info box to selected position
+			infoBox.sprite->offset.y = infoBoxOpts.yCenter;
+			infoBox.sprite->offset.x = infoBoxOpts.xCenter * 0.5f;

```

Comment:
I do not understand why dividing the `xCenter` position like this is necessary, only that the bounds for the infobox extend from `-0.25f to 0.25f` instead of `-0.5f to 0.5f` for some reason.

Happy to change this if you can you can point me in the right direction.